sherpa-onnx/csrc/offline-lm-config.cc

```diff
@@ -18,6 +18,8 @@ void OfflineLMConfig::Register(ParseOptions *po) {
                "Number of threads to run the neural network of LM model");
   po->Register("lm-provider", &lm_provider,
                "Specify a provider to LM model use: cpu, cuda, coreml");
+  po->Register("lodr-fst", &lodr_fst, "Path to LODR FST model.");
```

Comment:
Can you validate that if `--lodr-fst` is given and is not empty, then it should exist on the disk.